### ISSUE TITLE
Multiplex event streams

### ISSUE THREAD

**Issue by no-glue** [colin-dumitru/F.js#41]
Add ability to multiplex several event streams.

For example

```
var streamOne = F.eventStream(objectOne, "mousedown") // dom element
var streamTwo = F.eventStream(objectTwo, "message") // web sockets
var multiplex = F.multiplexStream(objectOne, objectTwo) // throws error "Cannot read property then of undefined"
```

**Comment by colin-dumitru** [comment]
Seems there was a bug with event streams. I've submitted a change to fix the issue: https://github.com/colin-dumitru/F.js/commit/92c135a1591ebe2bb0d42b330a5fd39d3be8f046 . I've also released a new version, 0.5.2 with said fix.

And here is a code pen for your use case: http://codepen.io/colin-dumitru/pen/OPQGgy

``` JavaScript
var streamOne = F.eventStream(document, "mousedown");
var streamTwo = F.eventStream(document, "keydown");

var multiplex = F(F.multiplexStream(streamOne, streamTwo)).log().pullStream();
```

When you type something or click the empty document, the event will be logged in the console.

BTW, You were also calling `multiplexStream` with objectOne/objectTwo instead of streamOne/streamTwo.

Can you check that the issue is fixed?

**Comment by no-glue** [comment]
Looks good now
